client/packages/config/src/config.ts

```diff
@@ -35,7 +35,10 @@ export const Environment = {
   PRINT_LABEL_TEST: `${apiHost}/print/label-test`,
 
   // Feature flags
-  FEATURE_INVENTORY_ADJUSTMENTS,
+  FEATURE_INVENTORY_ADJUSTMENTS:
+    typeof FEATURE_INVENTORY_ADJUSTMENTS === 'undefined'
+      ? false
+      : FEATURE_INVENTORY_ADJUSTMENTS,
```

Comment:
Allows a default value to be applied to the feature flag property if it is undefined (ie properly "not defined" , not just has a value of "undefined")

Reply:
This allows passing of strange values to the env var, would it be more reliable to just wrap in `FEATURE_INVENTORY_ADJUSTMENTS: Boolean(FEATURE_INVENTORY_ADJUSTMENTS)` - that would always return a boolean value. 

or truthy? with `!!FEATURE_INVENTORY_ADJUSTMENTS`

otherwise, to keep the current but simplify:

```
FEATURE_INVENTORY_ADJUSTMENTS: FEATURE_INVENTORY_ADJUSTMENTS ?? false;
```

Reply:
We tried that initially, but it doesn't work. The problem is because of the `declare const FEATURE_INVENTORY_ADJUSTMENTS: boolean;` at the top -- it tells the compiler that `FEATURE_INVENTORY_ADJUSTMENTS` definitely exists. But when it doesn't (i.e. running storybook), it freaks out when referring to a non-existing value.

I just copied your solution earlier in the file (for `API_HOST`). We can check the type of a non-existing value, but any reference to the value itself doesn't work.

Another option is just to add the env value to the storybook webpack, but we didn't want to have to maintain it in two different places.

I'd like to have a more standardised way of doing this, but this is the quickest fix to the current way env vars are being accessed. Maybe we could use [dotenv](https://www.npmjs.com/package/dotenv) or something to have simpler access to environment variables, but that would require more discussion.